README.md

```diff
@@ -30,12 +33,12 @@ This gives you incredible flexibility to use individual components and enable [a
 npm install fiori-fundamentals --save
 ````
 
-> NOTE: We only distribute the [fonts, icons and CSS](https://github.com/SAP/fundamental/tree/develop/dist) and [the SASS source](https://github.com/SAP/fundamental/tree/develop/scss), not the full project or HTML for specific components.
+> NOTE: We only distribute the [fonts, icons and CSS](https://github.com/SAP/fundamental/tree/master/dist) and [the SASS source](https://github.com/SAP/fundamental/tree/master/scss), not the full project or HTML for specific components.
 
-# Working with the project
-You can clone the entire repo and work in the full project where we manage the [documentation](https://github.com/SAP/fundamental/tree/develop/docs) and the [test framework](https://github.com/SAP/fundamental/tree/develop/test).
+## Working with the Project
+You can clone the entire repo and work in the full project where we manage the [documentation](https://github.com/SAP/fundamental/tree/master/docs) and the [test framework](https://github.com/SAP/fundamental/tree/master/test).
 
-#### Download and Installation
+### Download and Installation
 
 1. **Clone Repository** - Clone the repo using the git software of your choice or using the git command `git clone https://github.com/SAP/fundamental.git`
 
```

Comment:
should not need to install gulp globally. it is already a dependency of this project.